packages/typeorm-legacy/README.md

```diff
@@ -7,43 +7,44 @@
    </p>
    <p align="center" style="align: center;">
       <img src="https://github.com/nextauthjs/adapters/actions/workflows/release.yml/badge.svg" alt="Canary CI Test" />
-      <img src="https://img.shields.io/bundlephobia/minzip/@next-auth/typeorm-legacy-adapter/canary" alt="Bundle Size"/>
-      <img src="https://img.shields.io/npm/v/@next-auth/typeorm-legacy-adapter/canary" alt="@next-auth/typeorm-legacy-adapter Version" />
+      <img src="https://img.shields.io/bundlephobia/minzip/@next-auth/typeorm-legacy-adapter/next" alt="Bundle Size"/>
+      <img src="https://img.shields.io/npm/v/@next-auth/typeorm-legacy-adapter/next" alt="@next-auth/typeorm-legacy-adapter Version" />
    </p>
 </p>
 
 ## Overview
 
 This is the TypeORM Adapter for [`next-auth`](https://next-auth.js.org). This package can only be used in conjunction with the primary `next-auth` package. It is not a standalone package.
 
-You can find more TypeORM information in the docs at [next-auth.js.org/adapters/typeorm/typeorm-overview](https://next-auth.js.org/adapters/typeorm/typeorm-overview).
+You can find more TypeORM information in the docs at [next-auth.js.org/adapters/typeorm](https://next-auth.js.org/adapters/typeorm).
 
 ## Getting Started
 
-1. Install `next-auth` and `@next-auth/typeorm-legacy-adapter@canary`
+1. Install `typeorm`, `next-auth@beta` and `@next-auth/typeorm-legacy-adapter@next`
 
 ```js
-npm install next-auth @next-auth/typeorm-legacy-adapter@canary
+npm install next-auth@beta @next-auth/typeorm-legacy-adapter@next typeorm
 ```
 
 2. Add this adapter to your `pages/api/[...nextauth].js` next-auth configuration object.
 
 ```js
 import NextAuth from "next-auth"
-import Providers from "next-auth/providers"
-import Adapters from "next-auth/adapters"
+import GoogleProvider from "next-auth/providers/google"
+import { TypeORMLegacyAdapter } from "@next-auth/typeorm-legacy-adapter"
+
 
 // For more information on each option (and a full list of options) go to
 // https://next-auth.js.org/configuration/options
 export default NextAuth({
   // https://next-auth.js.org/configuration/providers
   providers: [
-    Providers.Google({
+    GoogleProvider({
       clientId: process.env.GOOGLE_ID,
       clientSecret: process.env.GOOGLE_SECRET,
     }),
```

Comment:
I think we should just drop `providers` from READMEs and focus on the `adapter`. Most users probably won't use Google anyway, and the first thing they do after pasting is to remove it.

Reply:
Yeah sure thing, so no providers in the example code in the adapters?

Reply:
Yes. Don't think it is necessary